### DISCUSSION TITLE
Internal Services landing page - 'cards'

### DISCUSSION THREAD

**Discussion by htmlandbacon** [dwp/design-system-community-backlog#75] [Q&A]
I've seen this used a couple of places as a landing page for agent services.

<img width="528" alt="example of cards giving links to different options within the service" src="https://user-images.githubusercontent.com/4334015/139046374-80a22590-2bec-4757-b5c3-349ac84f6669.png">

Slight variation of the actual card - the whole box is clickable/more of a border.

This version is taken from NHS: https://service-manual.nhs.uk/design-system/components/card#Clickable%20card

<img width="497" alt="variation of card item with drop shadow" src="https://user-images.githubusercontent.com/4334015/139047193-e2edb006-1098-4a70-81f4-05d9624f4082.png">

**Comment by gazjoy** [comment]
On JSA we have a landing screen with some blue boxes to show tasks. Each of the tasks are visible to an agent of a different tier. Some users see a few, others can see all of them. This was never intended to be the thing that got built, but here we are and we haven't had any negative feedback as far as I know.

![image](https://user-images.githubusercontent.com/628064/139096241-584694fd-9f1d-43f6-bbbf-cc8d84b306db.png)

I don't see how much value the blue boxes add over a simple form with radios though:
![image](https://user-images.githubusercontent.com/628064/139097681-3cf717cc-ff06-49a7-a007-92101da7945f.png)

**Reply by gazjoy** [reply]
An assumption on the cards is that if you make the whole thing clickable it is difficult for users to copy and paste any content inside the link tag with a mouse. So be mindful of what the card contains and its purpose.

**Reply by martinwake** [reply]
Fully clickable 'cards' might be easier to use than radios on a phone / other touch interface, perhaps? That may not be relevant for agent facing applications though.

**Comment by a184studio** [comment]
We didn't find any need for cards, we started with the humble list and has no evidence that it needed escalating. 

We had credential based access also, dynamic to role or location. 
 

Also we tried to make agents tell us what to use for a H1, for bit of stimulus we put My home page. Annoyingly, it stuck. 

However MyHomePage was useful for navigating back here from within an action or task. As the word "home" causes chaos with screen readers.    

We went left to right to mirror the GOV.uk menu style layout so it was consistant when switching task quickly (This part was never deployed)

So arguably this should all aligned left and stacked. 

Home
<img width="1127" alt="Screen Shot 2023-03-17 at 14 54 43" src="https://user-images.githubusercontent.com/44495654/225944752-dca44a25-4b39-4990-8764-c0caabeb2125.png">

In a task
<img width="1123" alt="Screen Shot 2023-03-17 at 15 07 51" src="https://user-images.githubusercontent.com/44495654/225944748-04f8226e-3485-4c4d-a75e-6b857e2c40cd.png">
